OrbitLinuxTracing/include/OrbitLinuxTracing/TracerListener.h

```diff
@@ -21,6 +21,7 @@ class TracerListener {
   virtual void OnThreadStateSlice(orbit_grpc_protos::ThreadStateSlice thread_state_slice) = 0;
   virtual void OnAddressInfo(orbit_grpc_protos::AddressInfo address_info) = 0;
   virtual void OnTracepointEvent(orbit_grpc_protos::TracepointEvent tracepoint_event) = 0;
+  virtual void OnMapsUpdate(uint64_t timestamp, const std::string& maps) = 0;
```

Comment:
Notice how this method is so inconsistent compared to all the others. They all accept protobufs from `orbit_grpc_protos` and this doesn't. I think that this should take `ModulesUpdateEvent` (maybe renamed to `MapsUpdateEvent`) and that should carry unparsed `maps`, but then those maps should be parsed anyway on the service and the protobuf translated. So more easily (and see the other comment), it could carry maps already parsed in `OrbitLinuxTracing`.

Reply:
as discussed offline - moved some code around to be able to parse maps in OrbitLinuxTracing. Note that first commit is PR#1558